package.json

```diff
@@ -71,7 +72,7 @@
     "eslint": "^3.12.0",
     "eslint-config-airbnb": "^13.0.0",
     "eslint-plugin-import": "^2.2.0",
-    "eslint-plugin-jsx-a11y": "^2.2.3",
```

Comment:
`2.2.3` was the minimum dependency for `airbnb` didn't wanted to move to v3 in case of braking changes.